pages/04.applications/99.wishlist/apps_wishlist.md

```diff
@@ -161,6 +162,7 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [Koozip](https://koozic.net) |  | [Upstream](https://github.com/DocMarty84/koozic) |  |
 | L'atelier |  |  [Upstream](https://github.com/jbl2024/latelier) |  |
 | [LBCAlerte](https://alerte.ilatumi.org/) |  |  | [Package Draft](https://github.com/YunoHost-Apps/LBCAlerte_ynh) |
+| [Leantime](https://leantime.io/) | Project Management for the non-project manager | [Upstream](https://github.com/Leantime/leantime) |  |
```

Comment:
Leantime is packaged (not working yet)
https://github.com/YunoHost-Apps/leantime_ynh

as well as Froxlor  (not working yet)
https://github.com/YunoHost-Apps/froxlor_ynh

Reply:
Wow, that was lightning fast. Thank you so much! I attempted to install both web apps with custom My Webapp which succeeded for Leantime with the custom redirect rules from this thread: https://forum.yunohost.org/t/my-webapp-pointing-domain-to-public-subfolder/24588/4 

For Froxlor I ran into a major issue when running the configuration of services (including nginx configuration). The service configuration broke my nginx service (see here: https://forum.yunohost.org/t/configuration-is-broken-for-service-nginx-unknown-directive-access-by-lua-file/24598/5 ) and unfortunately, I didn't have the capacities to debug it to the root cause but reinstalled Yunohost as I needed urgent access to some of my apps. Just to keep this issue in mind when packaging.

If I can help you with packaging in any way let me know.